doc/sphinxext/credit_tools.py

```diff
@@ -26,26 +26,68 @@
 doc_root = repo_root / "doc"
 data_dir = doc_root / "sphinxext"
 
+# TODO: For contributor names there are three sources of potential truth:
+#
+# 1. names.inc
+# 2. GitHub profile names (that we pull dynamically here)
+# 3. commit history / .mailmap.
+#
+# All three names can mismatch. Currently we try to defer to names.inc since this
+# is assumed to have been chosen the most consciously/intentionally by contributors.
+# Though it is possible that people can change their preferred names as well, so
+# preferring GitHub profile info (when complete!) is probably preferable.
+
+# Allowed singletons
+single_names = "btkcodedev buildqa sviter Akshay".split()
+# Allowed abbrevitaions in first/last name
+abbreviations = ["T. Wang"]
+# Manual renames
+manual_renames = {
+    "alexandra": "Alexandra Corneyllie",  # 7600
+    "alexandra.corneyllie": "Alexandra Corneyllie",  # 7600
+    "akshay0724": "Akshay",  # 4046, TODO: Check singleton
+    "AnneSo": "Anne-Sophie Dubarry",  # 4910
+    "Basile": "Basile Pinsard",  # 1791
+    "ChristinaZhao": "Christina Zhao",  # 9075
+    "Drew, J.": "Jordan Drew",  # 10861
+    "Hamid": "Hamid Maymandi",  # 10849
+    "jwelzel": "Julius Welzel",  # 11118
+    "Jean-Rémi King": "Jean-Rémi King",
```

Comment:
this one looks vacuous, maybe a "find/replace-all" mistake?

Reply:
Yep, removed!